datarepo-clienttests/src/main/java/testscripts/KubeConfig.java

```diff
@@ -0,0 +1,177 @@
+package testscripts;
+
+import bio.terra.datarepo.api.RepositoryApi;
+import bio.terra.datarepo.api.ResourcesApi;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.model.*;
+import io.kubernetes.client.openapi.models.V1Deployment;
+import io.kubernetes.client.openapi.models.V1Pod;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import utils.DataRepoUtils;
+import utils.FileUtils;
+import utils.KubernetesClientUtils;
+
+public class KubeConfig extends runner.TestScript {
+
+  /** Public constructor so that this class can be instantiated via reflection. */
+  public KubeConfig() {
+    super();
+  }
+
+  private int filesToLoad;
+  private String datasetCreator;
+  private BillingProfileModel billingProfileModel;
+  private DatasetSummaryModel datasetSummaryModel;
+
+  public void setParameters(List<String> parameters) throws Exception {
+    if (parameters == null || parameters.size() == 0) {
+      throw new IllegalArgumentException(
+          "Must provide a number of files to load between 1 and 25 in the parameters list");
+    } else {
+      // filesToLoad should be between 1 and 25. Default to 25.
+      int filesToLoadParam = Integer.parseInt(parameters.get(0));
+      filesToLoad = filesToLoadParam <= 25 && filesToLoadParam > 0 ? filesToLoadParam : 25;
```

Comment:
I think this is fine for now. As we understand the capabilities of a service instance we will want to be able to move many many more files in this. We do not need any additional source files. We can we reuse them. We would need to make more target files using the same source piles.

Reply:
Ok - I think I've made it so we can resuse them. Thanks!